vividus-plugin-mobile-app/build.gradle

```diff
@@ -9,6 +9,7 @@ project.description = 'Vividus plugin for testing mobile applications'
 
      implementation(group: 'org.slf4j', name: 'slf4j-api', version: versions.slf4j)
      implementation(group: 'javax.inject', name: 'javax.inject', version: versions.javaxInject)
+     implementation(group: 'org.boofcv', name: 'boofcv-core', version: '0.40.1')
```

Comment:
Shouldn't we have it as an extension to the visual testing plugin?

Reply:
+ If we want to make the strategy standalone then we will also have to move things out from plugin mobile like touch actions etc into separate plugin, so we will have two more modules for what?
+ 254.7 with boofcv vs 241 without boofcv doesn't seem like a big win